bin/publish_helm_chart

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -evuo pipefail
+
+if [ $PUBLISH_HELM_CHART != true ]; then
+    echo "publish flag not set"
+    exit 1
```

Comment:
Is this an error, or should we actually set zero as the exit code here?

Reply:
This script should just be run from the build server where the flag is set in global env.

Come to think of it now I added this to prevent running git push using my own credentials while extending the script. I think I'll just remove it as we are now picking up credentials from the environment.